src/main/target/REVO/target.h

```diff
@@ -17,20 +17,33 @@
 
 #pragma once
 
-#define TARGET_BOARD_IDENTIFIER "REVO"
-
 #define CONFIG_START_FLASH_ADDRESS (0x08080000) //0x08080000 to 0x080A0000 (FLASH_Sector_8)
 
+#if defined(AIRBOTF4)
+#define TARGET_BOARD_IDENTIFIER "AIR4"
+#define USBD_PRODUCT_STRING     "AirbotF4"
+
+#elif defined(REVOLT)
+#define TARGET_BOARD_IDENTIFIER "RVLT"
+#define USBD_PRODUCT_STRING     "Revolt"
+
+#else
+#define TARGET_BOARD_IDENTIFIER "REVO"
 #define USBD_PRODUCT_STRING     "Revolution"
+
 #ifdef OPBL
 #define USBD_SERIALNUMBER_STRING "0x8020000"
 #endif
 
+#endif
+
 #define USE_DSHOT
 
 #define LED0                    PB5
 // Disable LED1, conflicts with AirbotF4/Flip32F4 beeper
-//#define LED1                    PB4
+#if !defined(AIRBOTF4)
+#define LED1                    PB4
+#endif
 
 #define BEEPER                  PB4
```

Comment:
The BEEPER definition should be conditional on AIRBOTF4 to avoid LED1 conflict on REVO. Maybe an #else branch on LED1 condition above, since PB4 should be either LED1 or BEEPER. Not both.